pl/tutorials/tour/case-classes.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: tutorial
+title: Klasy case
+
+disqus: true
+
+tutorial: scala-tour
+num: 10
+language: pl
+tutorial-next: pattern-matching
+tutorial-previous: currying
+---
+
+Scala wspiera mechanizm _klas case_. Klasy case są zwykłymi klasami z dodatkowymi założeniami:
+
+* Domyślnie immutable
```

Comment:
`immutable` or `niezmienny` seems to be commonly used on Polish forums, but `niemutowalny` sounds very nice, I think we can use that instead.